tests/flit_makefile/tst_incremental_build.py

```diff
@@ -177,7 +177,7 @@
 ['gtrun', 'obj/gt/ALL-FLIT.cpp.o', 'obj/gt/Empty.cpp.o', 'obj/gt/main.cpp.o']
 
 >>> touched_files(after_build)
-[]
```

Comment:
Please insert a comment before this why this is needed and the issue number for fixing it.

Reply:
Done @mikebentley15, thanks for checking!

Let me know if you have any other suggestions.